src/ImmersedBoundaries/partial_cell_bottom.jl

```diff
@@ -150,55 +153,55 @@ end
     return !immersed_cell(i, j, k, grid, ib) & immersed_cell(i, j, k-1, grid, ib)
 end
 
-@inline function Δzᶜᶜᶜ(i, j, k, ibg::PCBIBG)
+@inline function Δrᶜᶜᶜ(i, j, k, ibg::PCBIBG)
```

Comment:
did you mean to change these?

Reply:
yeah, I think we need to change these, then the `Δzᶜᶜᶜ` is calculated as `Δrᶜᶜᶜ * dzdr`

Reply:
interesting I see